roles/satellite-clone/tasks/main.yml

```diff
@@ -1,17 +1,22 @@
 ---
-- include_vars: main.yml
-
 # include pre_install_check
 - include: pre_install_check.yml
   when: run_pre_install_check
 
+- name: "check that variables are updated"
```

Comment:
My thought behind putting it here instead of pre_install_check is that the pre_install_check can be skipped and this is a step that cannot be skipped (and is also easy to skip if a user just copies the file over w/o updating)